presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -248,6 +248,7 @@ class SystemConfig : public ConfigBase {
   /// Memory allocation limit enforced via internal memory allocator.
   static constexpr std::string_view kSystemMemoryGb{"system-memory-gb"};
 
+  static constexpr std::string_view kNativeSidecar{"native-sidecar"};
```

Comment:
Please can you add a comment explaining this config.

Reply:
Added a comment, thanks.

Reply:
@pramodsatya : Curious: Is this meant to be "can function" as a sidecar, or "is a" sidecar ?

Reply:
@aditi-pandit the plan is to register only the sidecar specific endpoints when this flag is enabled on the worker. Can I reword it as "Indicates if the worker is configured as a sidecar." ?

Reply:
@pramodsatya : That sounds reasonable.

Reply:
"Indicates if the process is configured as a sidecar."

Reply:
Updated, thanks.